src/Randomizer.App/ViewModels/TrackerMapLocationViewModel.cs

```diff
@@ -199,7 +252,24 @@ public ImageSource NumberImage
         /// Get the tag to use for the location. Use the region for dungeons
         ///  and the list of locations for all other places
         /// </summary>
-        public object Tag => Region.Name == Name ? Region : Locations.Where(x => Syncer.IsLocationClearable(x, true, Syncer.Tracker.World.Config.Keysanity)).ToList();
+        public object Tag
+        {
+            get
+            {
```

Comment:
I think this could be more readable as a switch expression

Reply:
I don't have a common type here to switch off of, unless I'm misunderstanding you (or am unfamiliar with something fancy in C#)